src/mkdocstrings/handlers/python.py

```diff
@@ -95,8 +95,11 @@ def render(self, data: CollectorItem, config: dict) -> str:  # noqa: D102 (ignor
             **{"config": final_config, data["category"]: data, "heading_level": heading_level, "root": True},
         )
 
-    def get_anchor(self, data: CollectorItem) -> str:  # noqa: D102 (ignore missing docstring)
-        return data.get("path")
+    def get_anchors(self, data: CollectorItem) -> Sequence[str]:  # noqa: D102 (ignore missing docstring)
+        try:
+            return (data["path"],)
+        except KeyError:
+            return ()
```

Comment:
```suggestion
        return data.get("path", ())
```

because exceptions are costly

Reply:
But that would return the path directly, without being wrapped in a tuple :confused:

Reply:
Oops nevermind then